lib/Doctrine/Template/Searchable.php

```diff
@@ -32,16 +32,19 @@
  */
 class Doctrine_Template_Searchable extends Doctrine_Template
 {
+    /** @var Doctrine_Search **/
+    protected $_plugin;
```

Comment:
`Doctrine_Search` defines some methods that `Doctrine_Record_Generator` doesn't (which is what this property is defined as in `Doctrine_Template`). This property is explicitly set to a `Doctrine_Search` object on line 47 below.